### PR TITLE
Biconomy Modular SDK V3

### PR DESCRIPTION
**Pull request by livingrockrises**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/web3-auth-native/package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
   },
   "peerDependencies": {
     "@babel/runtime": "^7.x",
-    "react-native": "*"
+    "react-native": "^3.0.0-alpha.0"
```
Comment (AmanRaj1608) [inline packages/web3-auth-native/package.json:46]: this is incorrect, needs to be updated

File: .gitignore [hunk 2]
```diff
@@ -59,14 +59,13 @@ deployments
...
+packages/account/package-lock.json
 packages/common/package-lock.json
-packages/ethers-lib/package-lock.json
+packages/bundler/package-lock.json
 packages/node-client/package-lock.json
-packages/relayer/package-lock.json
-packages/smart-account/package-lock.json
-packages/transactions/package-lock.json
+packages/paymaster/package-lock.json
 package-lock.json
```
Comment (AmanRaj1608) [inline .gitignore:67]: can add particle and web3-auth package lock files to ignore

File: README.md [hunk 3]
```diff
@@ -1,4 +1,47 @@
...
+
+
+
+## Architecture
+
+
+
+
+## Resources
+
```
Comment (AmanRaj1608) [inline README.md:43]: needs to be updated

**Review by AmanRaj1608 — COMMENTED** [review]
Need to fix the web3Auth react native dependency otherwise it will break for all users

File: packages/web3-auth-native/package.json (hunk 1 above)
Comment (livingrockrises) [reply packages/web3-auth-native/package.json:46]: nice catch. updating..

Comment (livingrockrises) [reply packages/web3-auth-native/package.json:46]: done with "~0.68.2"

File: .gitignore (hunk 2 above)
Comment (livingrockrises) [reply .gitignore:67]: done

**Review by AmanRaj1608 — APPROVED** [review]

**Review by talhamalik883 — APPROVED** [review]

**Pull request merged**